sources/packages/backend/libs/sims-db/src/entities/education-program.model.ts

```diff
@@ -66,12 +66,20 @@ export class EducationProgram extends RecordDataModel {
   })
   sabcCode: string;
   /**
-   * Regulatory body ode. Which regulatory body has approved your delivery of this program.
+   * Regulatory body code. Which regulatory body has approved your delivery of this program.
    */
   @Column({
     name: "regulatory_body",
   })
   regulatoryBody: string;
+  /**
+   * Other regulatory body code in case it is not listed in the form.
```

Comment:
IMO the comments written for a db entity model can exclude UI specific contents like form and could just be same as the column comment in db.